js/main.js

```diff
@@ -222,6 +223,38 @@ Site.Gallery = {
   },
 };
 
+Site.News = {
+  $news: undefined,
```

Comment:
This is not needed. Also I thought we agreed to declare this kind of values inside `init()`